book/04-git-server/sections/protocols.asc

```diff
@@ -3,15 +3,21 @@
 Git can use four major protocols to transfer data: Local, HTTP, Secure Shell (SSH) and Git.
 Here we'll discuss what they are and in what basic circumstances you would want (or not want) to use them.
 
+[NOTE]
+====
+In addition to the above, both FTP and FTPS protocols are supported, but those protocols
+are inefficient and deprecated, and won't be discussed here.
+====
+
 ==== Local Protocol
 
 (((protocols, local)))
-The most basic is the _Local protocol_, in which the remote repository is in another directory on disk.
+The most basic is the _Local protocol_, in which the remote repository is in another directory on the same disk.
```

Comment:
Well technically not, and the very next sentence will contradict this change. "Local clone" just means that Git is talking to the filesystem API to get the data from the remote repository, rather than network API. On my Mac I can do `cd /tmp; git clone /Volumes/mountedNetworkDisk/remote.git`. That'll use the "local" transport, but still technically result in network traffic.

I'm not sure the right way to convey all that in five words, but "on the same disk" doesn't quite do it. Maybe this:

> […] the remote repository is in another directory in the filesystem.

What do you think?

Reply:
I'm not crazy about that wording, since it comes back to what I asked about before -- the difference between being in the same filesystem (allowing hardlinks), and being in a different filesystem but on the same disk. I'll just revert this back to the original wording for now, and ponder further.